### DISCUSSION TITLE
Shelly via cloud

### DISCUSSION THREAD

**Discussion by PaulusThe1** [indomus/forum#3057] [GENERALE - Discussioni generiche]
Ho questa problematica:
Ho 2 reti differenti in 2 abitazioni differenti (casa A e casa B), nella A ho il server su cui gira Home Assistant e la maggior parte dei miei Shelly configurati con l'integrazione nativa di H.A. Nella casa B sto installando altri shelly che vorrei raggiungere via cloud (essendo una rete diversa, l'integrazione nativa non li vede)
Come posso fare? Esiste un'integrazione Shelly che vede solo i dispositivi via cloud?

**Comment by snakuzzo** [comment]
Potresti usare il custom component Shelly4hass -> https://indomus.it/guide/integrare-componenti-shelly-a-home-assistant-via-shellyforhass/

... Ma devi considerare che da un pezzo non è più manutenuto. Puoi farlo sfruttando le api messe a disposizione da alterco e creare entità di tipo rest.
In alternativa, potresti pensare di esporre un broker mqtt sulla casa A e fare collegare quegli shelly a quel broker...creando di conseguenza apposite entità mqtt
